src/VolleyManagement.Backend/VolleyManagement.Services/TournamentService.cs

```diff
@@ -168,7 +168,11 @@ public List<Tournament> GetFinished()
         /// <returns>Tournament teams</returns>
         public List<TeamTournamentDto> GetAllTournamentTeams(int tournamentId)
         {
-            return _tournamentTeamsQuery.Execute(new FindByTournamentIdCriteria { TournamentId = tournamentId });
+            var result = _tournamentTeamsQuery.Execute(new FindByTournamentIdCriteria { TournamentId = tournamentId });
+
+            result?.ForEach(t => t.GroupName = "You won't catch me!");
```

Comment:
Thank you very much!

Reply:
After your changes - all tests still passed. And it is bad.
But after we add this line - GetAllTournamentTeams_TeamsExist_TeamsReturned() failed.

![image](https://user-images.githubusercontent.com/28676005/36793835-29e9213c-1ca7-11e8-93b2-10210ee6ac89.png)


Should we create any other test. If yes, can you describe case we must to test.

Reply:
I did work for you and counted all places - there are 2 more places which should be tested